src/main/java/org/mitre/synthea/helpers/Utilities.java

```diff
@@ -638,4 +642,29 @@ public static Map cleanMap(Map input) {
     });
     return input;
   }
+
+  /**
+   * Enable reading the given URI from within a JAR file.
+   * For example, for command-line args which may refer to internal or external paths.
+   * Note that it's not always possible to know when a user-provided path
+   * is within a JAR file, so this method should be called if it is possible the
+   * path refers to an internal location.
+   * @param uri URI to be accessed
+   */
+  public static void fixPathFromJar(URI uri) throws IOException {
```

Comment:
I know that you're just moving this method, but I don't think the name of it is particularly clear.

It's not doing anything with the Path. It's just letting the JVM know about another place it can look for files. Maybe something like `updateReadableLocations`?

Reply:
Updated to `enableReadingURIFromJar` and rebased to fix the PulseOx test failure